### ISSUE TITLE
wicked greater dragon's confusion damage calc

### ISSUE THREAD

**Issue by jshepler** [Silberdrachi/CanIIdle#17]
Confusion appears to use the player's base max hit (before combat triangle). In this particular example, I'm using melee, base max hit is 891, after combat triangle is 688.

688 x .75 x .7 = 361.2; CII shows 362

891 x .75 x .7 = 467.775; game shows 467:

![image](https://user-images.githubusercontent.com/1145213/154846684-3e282b89-5ced-4f6d-bab5-a75ddc852e60.png)

CII should not be asking for max hit vs magic, it should be asking for base max hit. Besides, it already knows which combat style being used, it could have done the combat triangle calc itself instead of making the user do it.

I haven't tested if this applies to other similar attacks, e.g. furious horned elite's horn shots attack.

**Comment by jshepler** [comment]
I tested FHE's horn shots and it looks like it uses the FHE's max hit and not the player's, even though the descriptions for both horn shots and confusion say "enemy".

**Comment by jshepler** [comment]
You might want to consider removing this feature altogether and just have the same kind of warning you have for greater skeleton dragon that it's unable to determine with certainty due to various modifiers in the game that would have an effect. For example, the bonus from slayer crossbow or the astrology modifiers to damage done to slayer area and slayer task targets - the effects from such modifiers wouldn't be displayed while not attacking anything and so the player would not be able to accurately determine their max hit before combat triangle.

Some discussion about this on reddit: https://www.reddit.com/r/MelvorIdle/comments/sx3wbm/can_i_idle_wicked_greater_dragon_question/

**Comment by Silberdrachi** [comment]
Yep, after digging into it for a while theres no good way to fix it without having way too much information for the user to input (some thats  not even totally visible to the user). And like suggested i did just pull it out. Confusion is now hardcoded to 0 Damage, the warning text has been updated, and the highlighting will stay enabled always. I also just pulled out the player max hit field from the UI in general as its no longer needed.

Thanks for the report.
